hack/dockerfile/install-binaries.sh

```diff
@@ -47,11 +47,15 @@ install_proxy() {
 }
 
 install_dockercli() {
-	echo "Install docker/cli version $DOCKERCLI_COMMIT"
-	git clone "$DOCKERCLI_REPO" "$GOPATH/src/github.com/docker/cli"
-	cd "$GOPATH/src/github.com/docker/cli"
-	git checkout -q "$DOCKERCLI_COMMIT"
-	go build -o /usr/local/bin/docker github.com/docker/cli/cmd/docker
+	DOCKERCLI_CHANNEL=${DOCKERCLI_CHANNEL:-edge}
+	DOCKERCLI_VERSION=${DOCKERCLI_VERSION:-17.06.0-ce}
+	echo "Install docker/cli version $DOCKERCLI_VERSION from $DOCKERCLI_CHANNEL"
+
+	url=https://download.docker.com/linux/static
+	curl -Ls $url/$DOCKERCLI_CHANNEL/x86_64/docker-$DOCKERCLI_VERSION.tgz | \
```

Comment:
The release branches and version tags are in https://github.com/docker/docker-ce. Since the release branch receives cherry-picked patches there is unfortunately no corresponding commit in `docker/cli`.